testing/describes.js

```diff
@@ -615,7 +615,7 @@ class RealWinFixture {
         resolve();
       };
       iframe.onerror = reject;
-      document.body.appendChild(iframe);
+      document.body.insertBefore(iframe, document.body.firstChild);
```

Comment:
**Curious:** I've noticed that some tests replace the Karma debugging iframe with the test iframe. Will this prevent that from happening?

![image](https://user-images.githubusercontent.com/26553114/61820316-baf12e00-ae22-11e9-84ea-64317166dbe2.png)